src/os/osexec.go

```diff
@@ -0,0 +1,56 @@
+//go:build linux && !baremetal && !darwin && !tinygo.wasm && !arm64
```

Comment:
```suggestion
//go:build linux && !baremetal && !tinygo.wasm && !arm64
```

It can't be both linux and darwin.